test/dialect/AIE/badconnect.mlir

```diff
@@ -9,7 +9,7 @@
 //===----------------------------------------------------------------------===//
 
 // RUN: not %PYTHON aiecc.py %s 2>&1 | FileCheck %s
-// CHECK: error{{.*}} 'AIE.connect' op source index cannot be less than zero
+// CHECK: error{{.*}} 'AIE.connect' op attribute 'sourceChannel' failed to satisfy constraint: 32-bit signless integer attribute whose minimum value is 0
```

Comment:
This is odd... why did this change?

Reply:
Because it's 

> This is odd... why did this change?

Odd indeed. I've checked the history of those constraints (`ConfinedAttr` and `IntMinValue`), and I don't understand how the message could have been anything else 😕

Reply:
Simple: if you change to `uint32_t` everywhere then

https://github.com/Xilinx/mlir-aie/blob/dd2bf4899889a92d4c93742e9cdb67612d87067b/lib/Dialect/AIE/IR/AIEDialect.cpp#L943

is never false but the constraint on the input still trips (which it does anyway [tested by deleted the check]).